Doc/library/signal.rst

```diff
@@ -503,6 +503,21 @@ be sent, and the handler raises an exception. ::
 
    signal.alarm(0)          # Disable the alarm
 
+:class:`enums <enum.IntEnum>` types can be used to convert from signal code to string, or the reverse::
```

Comment:
I think there's no need to duplicate the first example here. We can enhance it instead of adding another one:

```py
import signal, os

def handler(signum, frame):
    signame = signal.Signals(signum).name
    print(f'Signal handler called with signal {signame} ({signum})')
    raise OSError("Couldn't open device!")

# Set the signal handler and a 5-second alarm
signal.signal(signal.SIGALRM, handler)
# An enum member can also be passed as first argument
signal.signal(signal.Signals['SIGALRM'], handler)
signal.alarm(5)

# This open() may hang indefinitely
fd = os.open('/dev/ttyS0', os.O_RDWR)

signal.alarm(0)          # Disable the alarm
```

Reply:
Also, it would be nice to document enums at the documentation:

```rst
The signal module defines three enums:

.. class:: Signals

.. You may be able to use ``.. enum:: Signals`` instead. I don't know if it's supported now.

   Document it here.

   .. versionadded:: 3.5
```